parse/ParseImpl.h

```diff
@@ -126,9 +126,13 @@ namespace parse { namespace detail {
 
         TraceLogger() << "Parse: Elapsed time to parse " << path.string() << " = " << (timer.elapsed() * 1000.0);
 
-        std::ptrdiff_t distance = std::distance(first, parse::detail::s_end);
+        // s_end is global and static.  In is wrong when multiple files are concurrently or
```

Comment:
In is -> It is
This change looks like it should be committed to master immediately and independently of this PR